.github/workflows/js-tests.yml

```diff
@@ -5,7 +5,29 @@ on:
     branches:
       - develop
       - main
+    # Only run if CSS/JS/MD-related files changed.
+    paths:
+      - '.github/workflows/js-tests.yml'
+      - '**.js'
+      - '**.json'
+      - '**.scss'
```

Comment:
Why would JS tests be concerned with `scss` file changes? 🤔

Reply:
Ah, for us I guess they wouldn't. Probably in general they won't, good point 👍🏻